helpers/eventwebhook/eventwebhook.go

```diff
@@ -33,9 +33,9 @@ func NewSettings() *Settings {
 	return &Settings{}
 }
 
-// SetEnable ...
-func (s *Settings) SetEnable(enable bool) {
-	s.Enable = &enable
+// SetSecureWebhookEnable ...
+func (s *Settings) SetSecureWebhookEnable(enable bool) {
```

Comment:
could we call this `ToggleSignedWebhookRequests`?

Reply:
This is not toggling. User shouldn;t do it but user can call this function 5 times with `SetSecureWebhookEnable(true)`, and every time it would try to enable it. Do we want to make this toggle function without the input arg?